docs/Tutorials/Advanced/ticketing/drop.md

```diff
@@ -0,0 +1,390 @@
+``` js
+// Imports:
+const { initKeypom, createDrop, createNFTSeries, addToBalance } = require("keypom-js");
+const { KeyPair, keyStores, connect } = require("near-api-js");
+const { parseNearAmount } = require("near-api-js/lib/utils/format");
+const path = require("path");
+const homedir = require("os").homedir();
+
+
+async function createTickDrop(){
+// STEP 1: Initiate a NEAR connection.
+
+// STEP 2: Create the drop with funciton call data.
+
+// STEP 3: Make NFT series for POAPs.
+}
+
+createTickDrop()
+```
+
+## Getting Started
+In this section, you'll be addressing the first step: connecting to NEAR. 
+
+This is done with `NEAR-API-JS` and consists of:
+
+1) Create a Keystore, which stores your access keys used to sign transactions   
+  * select a network, either `testnet` or `mainnet`  
+  * choose a location where the keypairs live, either a folder on your local machine, or in-memory      
+
+2) Define a NEAR configuration using the Keystore  
+3) Use the configuration to initialize a connection to NEAR  
+
+More information about this process can be found [here](https://docs.near.org/tools/near-api-js/quick-reference#key-store).
+
+:::note
+For simplicity, this tutorial will choose a file-based keystore and point to the `~/.near-credentials` folder on your local machine since this is where most of your keys are stored. For more information about KeyStores, visit NEAR's [official docs](https://docs.near.org/tools/near-api-js/quick-reference#key-store).
+:::
+
+```js reference
+https://github.com/keypom/keypom-js/blob/5e4b4744a16c727d96d235282020c186edd0b0b5/docs-advanced-tutorials/ticket-app/frontend/utils/createTickDrop.js#L9-L26
+```
+
+---
+
+## Creating Drop with Function Call Data
+In this section, you'll create the function call drop to meet the functional requirements defined earlier.
+
+Recall that the drop needs the following properties:
+
+* The drop must have 2 key uses for each key.
+* The second key use should call `nft_mint`.
+* the first key use must be password protected.
+
+### Function Call Drop Basics
+This process starts with calling the `initKeypom` function and will always be the first function you call to interact with the SDK. 
+
+`initKeypom` initializes the SDK to allow for interactions with the Keypom smart contracts. Without it, none of the other SDK functions would work as expected. If a NEAR connection is not already present, it will initialize a new one for you. More info on the `initKeypom` function can be found [here](../../../keypom-sdk/modules#initkeypom).
+
+After `initKeypom` is called, the FC Drop can be created by calling `createDrop` and adding an `fcData` parameter. 
+
+:::tip
+Recall that the private keys being generated using `createDrop` are used to store the assets. These keys are then embedded within a link.
+:::
+
+The primary task in creating the Function Call Drop is to define `fcData`. It is an object with the following properties.
+
+```bash
+fcData
+├── methods
```

Comment:
I'd expand on this a bit more:

fcData has methods for each key use. Key use 1 can have XYZ methods and key use 2 can have ABC methods etc... The methods in each use are executed sequentially.